lib/svgo/coa.js

```diff
@@ -359,12 +359,9 @@ function processSVGData(config, info, data, output, input) {
     prevFileSize = Buffer.byteLength(data, 'utf8');
 
   const result = optimize(data, { ...config, ...info });
-  if (result.error) {
-    let message = result.error;
-    if (result.path != null) {
-      message += `\nFile: ${result.path}`;
-    }
-    throw Error(message);
+  if (result.modernError) {
+    console.error(red(result.modernError.toString()));
+    return;
```

Comment:
Just wondering shouldn't you use `process.exit()` here?

Reply:
Good catch